src/libraries/System.Console/tests/SyncTextReader.cs

```diff
@@ -78,7 +78,7 @@ private static void Test(string content, Action action)
     }
 
     [Fact]
-    [SkipOnPlatform(TestPlatforms.Browser, "Not supported on Browser.")]
+    [SkipOnPlatform(TestPlatforms.Browser | TestPlatforms.iOS | TestPlatforms.tvOS, "Not supported on Browser, iOS, or tvOS.")]
```

Comment:
Perhaps, the entire SyncTextReader test class can be annotated for those three platforms instead of each method.